src/main/resources/resource/Arduino/MrlComm/MrlServo.cpp

```diff
@@ -137,16 +138,19 @@ void MrlServo::startSweep(int minUs, int maxUs, int step) {
   targetPosUs = maxUs;
   isMoving = true;
   isSweeping = true;
+  publishServoEvent(SERVO_EVENT_STARTED);
 }
 
 void MrlServo::stop() {
   isMoving = false;
   isSweeping = false;
+  publishServoEvent(SERVO_EVENT_STOPPED);
```

Comment:
the servo doesn't actually stop here.. the actual stop occurs in update()  .. avoid writing to the serial port from inside the processCommand method.